### DISCUSSION TITLE
Telegram community

### DISCUSSION THREAD

**Discussion by Githuber522** [FossifyOrg/General-Discussion#218] [Q&A]
I want to join the Telegram community.
 
 After the installatin  I can see only the logo, no chats.
 
 Is the chat stopped at the moment?

**Comment by naveensingh** [comment]
It's not a group, it's a channel.

> Is the chat stopped at the moment?

Yes.

**Reply by Githuber522** [reply]
ok, thanks.

**Reply by Githuber522** [reply]
so as a new member i even cannot see older articles in the channel right now?

**Reply by naveensingh** [reply]
No, nothing was ever posted on the channel.
